OktaAuthNative.xcodeproj/xcshareddata/xcschemes/OktaAuthNative iOS.xcscheme

```diff
@@ -47,16 +47,6 @@
          </BuildableReference>
       </CodeCoverageTargets>
       <Testables>
-         <TestableReference
```

Comment:
?

Reply:
Disable it for now since we want to pass a branch to Lululemon and ask them to test it. We have a follow-up ticket to fix all other remaining issues caused by this code change: https://oktainc.atlassian.net/browse/OKTA-326273